be.go

```diff
@@ -221,3 +222,37 @@ func (b *Be) Num() float64 {
 		return 0
 	}
 }
+
+func (b *Be) True() bool {
```

Comment:
@jasonkeene, what do you think about align it with the other methods and return `Be`?
I'm not sure about it, because it doesn't make sense to chain another method after calling `True` or `False`